tests/test_support/test_support.py

```diff
@@ -155,3 +161,342 @@ def test_refresh_trusted_advisor_check_cycle_continues_on_full_cycle():
 
     expected_none_response["status"]["status"].should.equal("none")
+
+
+@mock_support
+def test_support_case_is_closed():
+    """
+    On closing a case, the correct response is returned
+    """
+    client = boto3.client("support", "us-east-1")
+    create_case_response = client.create_case(
+        subject="test_subject",
+        serviceCode="test_service_code",
+        severityCode="test_severity_code",
+        categoryCode="test_category_code",
+        communicationBody="test_communication_body",
+        ccEmailAddresses=["test_email_cc",],
+        language="test_language",
+        issueType="test_issue_type",
+        attachmentSetId="test_attachment_set_id",
+    )
+    case_id = create_case_response["caseId"]
+
+    resolve_case_response = client.resolve_case(caseId=case_id)
+
+    possible_case_status = [
+        "opened",
+        "pending-customer-action",
+        "reopened",
+        "unassigned",
+        "resolved",
+        "work-in-progress",
+    ]
+    expected_initial_case = [resolve_case_response["initialCaseStatus"]]
+    expected_final_case = "resolved"
+
+    set(expected_initial_case).issubset(
+        possible_case_status
+    ).should.be.true and expected_final_case.should.equal("resolved")
+
+
+@mock_support
+def test_support_case_created():
+    """
+    On creating a support request its response contains a case ID
+    """
+
+    client = boto3.client("support", "us-east-1")
+    create_case_response = client.create_case(
+        subject="test_subject",
+        serviceCode="test_service_code",
+        severityCode="test_severity_code",
+        categoryCode="test_category_code",
+        communicationBody="test_communication_body",
+        ccEmailAddresses=["test_email_cc",],
+        language="test_language",
+        issueType="test_issue_type",
+        attachmentSetId="test_attachment_set_id",
+    )
+
+    len(create_case_response["caseId"]).should.equal(38)
+
+
+@mock_support
+def test_support_created_case_can_be_described():
+    """
+    On creating a support request it can be described
+    """
+
+    client = boto3.client("support", "us-east-1")
+    create_case_response = client.create_case(
+        subject="test_subject",
+        serviceCode="test_service_code",
+        severityCode="test_severity_code",
+        categoryCode="test_category_code",
+        communicationBody="test_communication_body",
+        ccEmailAddresses=["test_email_cc",],
+        language="test_language",
+        issueType="test_issue_type",
+        attachmentSetId="test_attachment_set_id",
+    )
+
+    case_id_list = create_case_response["caseId"]
+    describe_cases_response = client.describe_cases(
+        caseIdList=[case_id_list],
+        displayId="test_display_id",
+        afterTime="test_after_time",
+        beforeTime="test_before_time",
+        includeResolvedCases=True,
+        nextToken="test_next_token",
```

Comment:
Can we also have a test without nextToken/maxResults set? Just to verify that it's not required